mitodl/ocw-hugo-themes
# ocw-course not course

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/170

#### What's this PR do?
Recently in `ocw-studio`, we implemented the use of imported `WebsiteStarter` config files from the [`ocw-hugo-projects`](https://github.com/mitodl/ocw-hugo-projects) repo.  The name of the course site starter there is `ocw-course`, so courses imported into `ocw-studio` (and subsequently delivered by the API) are of type `ocw-course`.  Previously the `ocw-studio` import process used a starter checked into the `ocw-studio` repo simply called `course`.  This PR updates the API request on `home_course_cards.html` to specify the `ocw-course` type rather than `course` so that courses are returned in the results.

#### How should this be manually tested?
 - Read the readme before if you have never spun up `ocw-www` locally
 - Clone https://github.com/mitodl/ocw-www
 - In your `.env`, make sure `EXTERNAL_SITE_PATH` is set to your path to `ocw-www` and make sure `OCW_STUDIO_BASE_URL` is set to http://ocw-studio-rc.odl.mit.edu and `OCW_IMPORT_STARTER_SLUG` is set to `ocw-course`
 - Run `npm start`
 - Visit http://localhost:3000/ and ensure that courses are displayed in the new courses section, noting that images will not work because of the current `coursemedia` prefix being assigned
